pkgs/development/python-modules/pylint-quotes/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools
+, wheel
+, pylint
+}:
+
+buildPythonPackage rec {
+  pname = "pylint-quotes";
+  version = "0.2.3";
+  pyproject = true;
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-LWuz+ooaha86+KDKh1pxmsW823NcRXVihGmdgJwQnJU=";
+  };
+
+  nativeBuildInputs = [
```

```suggestion
  build-system = [
```
And `dependencies` for runtime dependencies.